src/components/backToTop.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <span
+    class="btn-wrap"
+  >
+    <AeButton
+      class="top-btn"
+      face="icon"
+      fill="primary"
+      @click="goOnTop"
+    >
+      <AeIcon name="left-more" />
+    </AeButton>
+  </span>
+</template>
+
+<script>
+import {
+  AeButton,
+  AeIcon
+} from '@aeternity/aepp-components-3'
+export default {
+  name: 'BackToTop',
+  components: {
+    AeButton,
+    AeIcon
+  },
+  data () {
+    return {
+      scrolled: 0
+    }
+  },
+  beforeMount () {
+    window.addEventListener('scroll', this.handleScroll)
+  },
+  beforeDestroy () {
+    window.removeEventListener('scroll', this.handleScroll)
+  },
+  methods: {
+    goOnTop () {
+      window.scrollTo({
+        top: 0,
+        left: 0,
+        behavior: 'smooth'
+      })
+    },
+    handleScroll () {
```

Comment:
@artem-rozhkov-dev-optima why do you need this method? What I understand it does, it simply gives the Y position, but I don't see anywhere where you're using this?